addons/account/models/account_move.py

```diff
@@ -3489,6 +3489,7 @@ class AccountMoveLine(models.Model):
         help='Utility field to express amount currency')
     account_id = fields.Many2one('account.account', string='Account',
         index=True, ondelete="cascade",
+        auto_join=True,
```

Comment:
Couldn't this allow to read fields of `account_id` by going through `account.move.line`?
I'm not sure if the access rights are always the same

Reply:
If we have access to lines, we usually have access to related `account.move`, not?